docs/_shared/rst_prolog.rst

```diff
@@ -23,6 +23,10 @@
 
 ..  |Office iOS| replace:: Office for iOS
 
+..  |Office iOS Android| replace:: Office for iOS and Android
```

Comment:
How about `Office Mobile` for this placeholder?

Reply:
Well, we have windows phone too.